src/datajudge/db_access.py

```diff
@@ -650,14 +651,12 @@ def get_column(
         selection = sa.select([column])
 
         if is_snowflake(engine):  # check if we have a snowflake cursor
-            snowflake_cursor = engine.connect().connection.cursor()
+            snowflake_cursor: SnowflakeCursor = engine.connect().connection.cursor()
 
             # note: this step requires pandas to be installed
```

Comment:
This comment is outdated I believe.

Reply:
yes

Reply:
Is it, though? Because currently, it requires `pyarrow` _and_ `pandas` because we get the SegFault otherwise. I'll change it to reflect that, though.

Reply:
I wrote it before I was aware of the segfault without pandas.